sqlite/sqlite-bundled/build.gradle

```diff
@@ -55,19 +58,23 @@ def SQLITE_COMPILE_FLAGS = [
         "-DSQLITE_THREADSAFE=2",
 ]
 
-// Copy SQLite sources into a build directory. Even though this is not needed, it will be
-// nice to have them there so that we can swap them with SQLite amalgamation for Github.
-// TODO(b/306669673): Support GitHub builds
-def copySqliteSourcesTask = tasks.register("copySqliteSources", Copy) {
-    from(new File(AndroidXConfig.getExternalProjectPath(project), "sqlite/dist/orig")) {
-        include "sqlite3.c"
-        include "sqlite3.h"
+// see PrepareSqliteSourcesTask.groovy for implementation details of the register method
+def prepareSqliteSourcesTask = registerPrepareSqliteSourcesTask(
```

Comment:
somewhere within this file we should have a comment explaining the license of sqlite that we are relicensing it here as apache 2.0

Reply:
added a note on licensing.